### ISSUE TITLE
Idea Collector: Beepberry Shield

### ISSUE THREAD

**Issue by jamesmunns** [tosc-rs/mnemos#121]
So, we've had some ideas to make a mnemos shield since the MQ-Pro doesn't check as many boxes as we'd originally hoped. Collecting possible options here:

* Use a Lichee-RV SOM, so have 2x m.2 sockets
    * Has SD-Card and USB-OTG port on the SOM
* Have a QSPI flash for bootloader and for persistent image before we have SD Card support
* Have an ESP32C3 for wifi
* Have a lot of breakout pins for development

There are a ton of other options that I don't think would be immediately useful and would add complexity to the board, but maybe @hawkw disagrees:

* Ethernet PHY/port
* USB-A host port
* Some kind of higher resolution display (mipi, hdmi) option

**Comment by hawkw** [comment]
I agree that we probably don't want Ethernet or HDMI on the board; an ethernet jack is gonna make the whole stack even taller, which feels kinda sad, and the MQ Pro already has mini-HDMI on the board...

**Comment by hawkw** [comment]
A Lichee RV-based breakout does seem like it could potentially stick out a bit less far than a MQ Pro and header/socket...I would assume the dual-M.2 thingy lets us access more pins from the SOC on the board than just the 40-pin Pi GPIO header?

**Comment by jamesmunns** [comment]
Yeah, I have to check the pinouts, but there's a ton more on the lichee-rv. It's like 2x67 pins total, not sure how many are power only

**Comment by jamesmunns** [comment]
![Screenshot 2023-07-02 at 04-29-31 Page11_PORTG - Lichee_RV_7001_Schematic pdf](https://github.com/tosc-rs/mnemos/assets/2097964/e0a0fc0b-e6d0-4db2-baa5-1e0bbddfa7dd)

**Comment by hawkw** [comment]
@jamesmunns suggested on matrix that we just put XIAO/QT Py headers on the breakout for mounting a C3 dev board, instead of actually adding a SMD esp32c3:
> oh! I could put a XIAO pad on it for the C3!
> that would actually be pretty nice, and reasonably sized, and very removable for testing
> that would actually be pretty nice, and reasonably sized, and very removable for testing

**Comment by hawkw** [comment]
also, future revisions of the PCB should include a switch or solder pads or something on the 5v rail — right now we've had to do PCB surgery to disable it so we can use USB for flashing.

**Comment by hawkw** [comment]
It would be cool if a future revision broke out the Pi header's UART pins into a FTDI or similar USB-serial guy, so that there's no need for an external dongle hanging off the side with dupont wires.

**Comment by hawkw** [comment]
We could probably cross off the QSPI flash since we do have SD boot working already. But, it could still be cool to have.
